content/vault/v1.19.x/content/partials/release-notes/change-summary/1_19.mdx

```diff
@@ -42,4 +42,5 @@ Found  | Fixed  | Workaround | Edition    | Issue
 1.19.0 | 1.19.3 | **Yes**    | All        | [Login/token renewal failures after group changes](/vault/docs/v1.19.x/updates/important-changes#group-writes)
 1.19.0 | 1.19.3 | Upgrade    | All        | [Unexpected DB static role rotations on upgrade](/vault/docs/v1.19.x/updates/important-changes#db-static-role-rotations)
 1.19.0 | 1.19.3 | Upgrade    | All        | [Unexpected LDAP static role rotations on upgrade](/vault/docs/v1.19.x/updates/important-changes#ldap-static-role-rotations)
-1.19.0 | 1.19.3 | **Yes**    | All        | [Unwanted secret rotation for DB and LDAP roles on restart](/vault/docs/v1.19.x/updates/important-changes#secret-rotate-on-restart)
\ No newline at end of file
+1.19.0 | 1.19.3 | **Yes**    | All        | [Unwanted secret rotation for DB and LDAP roles on restart](/vault/docs/v1.19.x/updates/important-changes#secret-rotate-on-restart)
+1.19.0 | 1.20.0 | **Yes**    | All        | [Counters API requires start time in OSS Vault](/vault/docs/v1.19.x/updates/important-changes#start-time-counters-api)
```

Comment:
```suggestion
1.19.0 | No | **Yes**    | All        | [Counters API requires start time in OSS Vault](/vault/docs/v1.19.x/updates/important-changes#start-time-counters-api)
```
Since the issue is not fixed for v1.19.x, the "Fixed" column should say "No".